sympy/utilities/_compilation/util.py

```diff
@@ -0,0 +1,270 @@
+# -*- coding: utf-8 -*-
+from __future__ import (absolute_import, division, print_function)
+
+from collections import namedtuple
+from contextlib import contextmanager
+from hashlib import sha256
+import glob
+import io
+import os
+import shutil
+import sys
+import tempfile
+
+
+class CompilationError(Exception):
+    pass
+
+
+class FileNotFoundError(IOError):
```

Comment:
This masks the built in FileNotFoundError in Python 3.

Reply:
Thanks, will use the built-in in Py3